src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -293,56 +305,64 @@ private void incomingPacketInfo(
         // Check if incoming bitrate estimate is valid, and if it
         // needs to be reset.
         long incomingBitrate_ =
-                incomingBitrate.getRate(arrivalTimeMs);
+            incomingBitrate.getRate(arrivalTimeMs);
+
+
         if (incomingBitrate_ != 0)
         {
             incomingBitrateInitialized = true;
-        } else if (incomingBitrateInitialized)
+        }
+        else if (incomingBitrateInitialized)
         {
             // Incoming bitrate had a previous valid value, but now not
             // enough data point are left within the current window.
```

Comment:
Thank @bbaldino for the comment. I think if we read it as when `incomingBitrate_ == 0` and `incomingBitrateInitialized == true`, then I think it makes sense to say the 

> Incoming bitrate had a previous valid value, but now not
    enough data point are left within the current window

Since this would mean that `incomingBitrate_ ` had a previous valid value thus making `incomingBitrateInitialized `true but now equals `zero ` which could mean that there aren't enough data points.
I think this translates to the logic flow for setting `incomingBitrateInitialized` and checking when `incomingBitrate_ ` is zero. Correct me if am wrong. Thanks.